### PR TITLE
Bulk add SIM Card

### PR DESCRIPTION
**Pull request by sekuranka**

### CODE REVIEW

File: src/ralph/sim_cards/admin.py [hunk 1]
```diff
@@ -43,6 +45,16 @@ class SIMCardAdmin(RalphAdmin):
         }),
     )
 
+    def get_multiadd_fields(self, obj=None):
+        multi_add_fields = [
+            {'field': 'card_number', 'allow_duplicates': False},
+            {'field': 'phone_number', 'allow_duplicates': False},
+            {'field': 'pin1', 'allow_duplicates': False},
```
Comment (szymi-) [inline src/ralph/sim_cards/admin.py]: allow_duplicates = True - for pin and for puk

Comment (sekuranka) [reply src/ralph/sim_cards/admin.py]: done

**Review by szymi- — DISMISSED** [review]

**Review by szymi- — DISMISSED** [review]

File: src/ralph/sim_cards/admin.py [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
 from ralph.admin import RalphAdmin, register
+from ralph.admin.views.multiadd import MulitiAddAdminMixin
 from ralph.lib.transitions.admin import TransitionAdminMixin
 from ralph.sim_cards.forms import SIMCardForm
 from ralph.sim_cards.models import CellularCarrier, SIMCard, SIMCardFeatures
 
 
 @register(SIMCard)
-class SIMCardAdmin(TransitionAdminMixin, RalphAdmin):
+class SIMCardAdmin(MulitiAddAdminMixin, RalphAdmin, TransitionAdminMixin):
```
Comment (romcheg) [inline src/ralph/sim_cards/admin.py]: Better to put all mixins before the base class

```suggestion
class SIMCardAdmin(MulitiAddAdminMixin, TransitionAdminMixin, RalphAdmin):
```

**Review by romcheg — CHANGES_REQUESTED** [review]

**Review by MarekBleschke — DISMISSED** [review]

**Review by romcheg — APPROVED** [review]
My concerns have been resolved. LGTM, thanks!

**Pull request merged**